ckanext/xroad_integration/fanstatic/webassets.yml

```diff
@@ -9,6 +9,17 @@ xroad_stats_js:
     - javascript/libs/chartjs-datalabels.js
     - javascript/xroad_stats.js
 
+xroad_distinct_service_stats_js:
+  filters: rjsmin
+  output: ckanext-xroad_integration/%(version)s_xroad_distinct_service_stats.js
+  extra:
+    preload:
+      - base/main
+  contents:
+    - javascript/libs/Chart.js
+    - javascript/libs/chartjs-datalabels.js
```

Comment:
Now that there's two pages that need the chart libraries, we should split them into their own asset bundle and make both xroad_stats_js and xroad_distinct_service_stats_js preload it. see https://github.com/vrk-kpa/api-catalog/blob/master/ckanext/ckanext-apicatalog_ui/ckanext/apicatalog_ui/fanstatic/webassets.yml#L37-L65 for an example.